README.md

```diff
@@ -1,30 +1,39 @@
 # Conda documentation
 
-This repository includes documentation that is common for `conda` and `conda-build` (primarily landing pages at https://conda.io/).
+This repository includes documentation that is common for `conda` and `conda-build`, as well as landing pages for those projects (located at https://docs.conda.io/).
+
+The conda documentation is built using [ReadTheDocs](https://readthedocs.com/). The `conda-docs` repo is the primary project, while `conda` and `conda-build` are created as subprojects. This enables each project's respective documentation to remain in its repo, but for all of the documentation to exist together on the same domain. The landing pages built from the `conda-docs` repo link to the documentation built from the `conda` and `conda-build` repos.
```

Comment:
I started this paragraph initially to give guidance on how other conda-related projects could get their docs linked to the conda docs, but that's out of scope for the ticket, I realized. Also, not sure how we feel about highlighting how things work when we may be changing it.

Reply:
Oh, forgot to say that this _is_ useful, simply because it'll mean we can easily update it when we change the way we build this.

Reply:
Good points! 👍